src/main.c

```diff
@@ -152,6 +179,11 @@ int main(void) {
   Serial_Init(SERIAL_BAUDRATE);
   CAN_Init(CAN_BAUDRATE);
   ADC_Init();
+  Timer_Init();
+
+  Set_Interrupt_Priorities();
```

Comment:
Set_Interrupt_Priorities() also sets the prioroty of the SysTick interrupt. Since it sets priorities for the timer interrupts and the SysTick interrupt, I decided to put it in main.c